### PR TITLE
[Dialogs] Update description of adjustable insets.

### PR DESCRIPTION
**Pull request by galiak11**
# Description
Minor fixes to description of the private properties.

## Issue
Issues: b/148802180, 294022719

### CODE REVIEW

File: components/Dialogs/src/private/MDCAlertControllerView+Private.h [hunk 1]
```diff
@@ -56,69 +56,67 @@
 @property(nonatomic, assign) BOOL adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable;
 
 /**
- The margins around the title icon or the title icon view against the dialog
- edges (top, leading, trailing) and the title (bottom). Note that the actual
- bottom space is the smallest between titleImageInsets.bottom and
- titleInsets.top.
+ Weather adjustable insets mode is enabled for the dialog view. If set to @c
```
Comment (bryanoltman) [inline components/Dialogs/src/private/MDCAlertControllerView+Private.h]: Spelling – should be "Whether"

File: components/Dialogs/src/private/MDCAlertControllerView+Private.h [hunk 2]
```diff
@@ -56,69 +56,67 @@
...
+ YES, a new layout calculation that is customizable by the insets in this header
+ file is used to layout the dialog. If set to @c NO, we fall back to the legaccy
```
Comment (bryanoltman) [inline components/Dialogs/src/private/MDCAlertControllerView+Private.h]: Spelling – should be "legacy"

File: components/Dialogs/src/private/MDCAlertControllerView+Private.h [hunk 3]
```diff
@@ -56,69 +56,67 @@
...
+ layout calculation, ignoring all customized inset values.
+
+ Default value is @c NO.
+ */
+@property(nonatomic, assign) BOOL enableAdjustableInsets;
+
+/**
+The edge insets around the title icon or title icon view against the dialog
```
Comment (bryanoltman) [inline components/Dialogs/src/private/MDCAlertControllerView+Private.h]: The indentation of this line doesn't match the other comments

File: components/Dialogs/src/private/MDCAlertControllerView+Private.h [hunk 4]
```diff
@@ -56,69 +56,67 @@
...
+ edges (top, leading, trailing) and the title (bottom). Note that
+ `titleIconInsets.bottom` overrides `titleInsets.top`.
```
Comment (bryanoltman) [inline components/Dialogs/src/private/MDCAlertControllerView+Private.h]: Given that "overrides" means something very specific in OO programming, I think I would change this to "`titleIconInsets.bottom` takes precedence over `titleInsets.top`"

File: components/Dialogs/src/private/MDCAlertControllerView+Private.h [hunk 5]
```diff
@@ -56,69 +56,67 @@
...
 @property(nonatomic, assign) UIEdgeInsets titleIconInsets;
 
 /**
- The margins around the title against the dialog frame and its neighbor elements.
- If a title icon is presented, the minimum titleInsets.top and titleIconInsets.bottom is used.
- If a message is presented, the minimum titleInsets.bottom and contentInsets.top is used.
- If there is no message, titleInsets.bottom is used.
+ The edge insets around the title against the dialog edges or its neighbor
+ elements. If either the title icon or title icon view is present, then
+ `titleIconInsets.bottom` overrides `titleInsets.top`. If there is no message,
```
Comment (bryanoltman) [inline components/Dialogs/src/private/MDCAlertControllerView+Private.h]: Same comment here re: "overrides"

**Review by bryanoltman — APPROVED** [review]
LGTM pending some minor copy edits

File: components/Dialogs/src/private/MDCAlertControllerView+Private.h (hunk 1 above)
Comment (galiak11) [reply components/Dialogs/src/private/MDCAlertControllerView+Private.h]: :)

File: components/Dialogs/src/private/MDCAlertControllerView+Private.h (hunk 2 above)
Comment (galiak11) [reply components/Dialogs/src/private/MDCAlertControllerView+Private.h]: Done

File: components/Dialogs/src/private/MDCAlertControllerView+Private.h (hunk 3 above)
Comment (galiak11) [reply components/Dialogs/src/private/MDCAlertControllerView+Private.h]: Done

File: components/Dialogs/src/private/MDCAlertControllerView+Private.h (hunk 4 above)
Comment (galiak11) [reply components/Dialogs/src/private/MDCAlertControllerView+Private.h]: Done

File: components/Dialogs/src/private/MDCAlertControllerView+Private.h (hunk 5 above)
Comment (galiak11) [reply components/Dialogs/src/private/MDCAlertControllerView+Private.h]: Done

**Pull request merged**